environment.yml

```diff
@@ -0,0 +1,11 @@
+name: dbcsep
+channels:
+    - conda-forge
```

Comment:
Are we using `conda` here instead of `pip`?

Reply:
Both environment instantiations are given by either `environment.yml` or `setup.cfg`. In case a user uses `pip`, the conda `.yml` is not actually used. However this requires that when updating build instructions, both approaches needs to be reviewed.